### PR TITLE
feat(schematic): fds-1688 add endpoints that return files

### PR DESCRIPTION
**Pull request by andrewelamb**
- fixes [FDS-1688]
- Adds endpoints that return csv files
- Please review 

1. all files in `libs/schematic/api-description/src/`
2. all files in `apps/schematic/api/schematic_api/controllers/`  except  `storage_controller.py`

[FDS-1688]: https://sagebionetworks.jira.com/browse/FDS-1688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GiaJordan — APPROVED** [review]
Approving based on review of #2500 with the expectation the currently running actions are successful

**Comment by tschaffter** [comment]
How is this PR different from #2500 (same title and description)

**Comment by tschaffter** [comment]
It may be because of this new branch protection rule: reviewers must approve the latest version of the code submitted. Here, you have made modification after @GiaJordan's review.

image

This is stated below. GitHub could do a better job at invalidating reviews.

image

File: apps/schematic/api/schematic_api/test/conftest.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def fixture_example_manifest_metadata() -> Generator:
     yield EXAMPLE_MANIFEST_METADATA
 
 
-TEST_SCHEMA_URL = "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld"  # pylint: disable=line-too-long
+TEST_SCHEMA_URL = "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/main/tests/data/example.model.jsonld"  # pylint: disable=line-too-long
```
Comment (andrewelamb) [inline apps/schematic/api/schematic_api/test/conftest.py:81]: @GiaJordan I added this change

**Comment by andrewelamb** [comment]
@GiaJordan Could you re-approve? I made on change to fix the schema used for tests from the develop to main version.

**Review by GiaJordan — APPROVED** [review]

**Pull request merged**